src/base/http/types.h

```diff
@@ -63,7 +63,7 @@ namespace Http
 
     const char CONTENT_TYPE_HTML[] = "text/html";
     const char CONTENT_TYPE_CSS[] = "text/css";
```

Comment:
I wonder if charset should be added to other text content types too?

Reply:
So far I haven't seen browsers complaining, we can add them when the warnings emerged.

Reply:
Ok.